src/node.cc

```diff
@@ -3597,27 +3596,31 @@ static void PrintHelp() {
          "                        (overrides NODE_ICU_DATA)\n"
 #if !defined(NODE_HAVE_SMALL_ICU)
          "                        note: linked-in ICU data is\n"
-         "                        present.\n"
+         "                        present\n"
 #endif
          "  --preserve-symlinks   preserve symbolic links when resolving\n"
-         "                        and caching modules.\n"
+         "                        and caching modules\n"
 #endif
          "\n"
          "Environment variables:\n"
-#ifdef _WIN32
-         "NODE_PATH                ';'-separated list of directories\n"
-#else
-         "NODE_PATH                ':'-separated list of directories\n"
-#endif
-         "                         prefixed to the module search path.\n"
+         "NODE_DEBUG               ','-separated list of core modules that\n"
+         "                         should print debug information\n"
          "NODE_DISABLE_COLORS      set to 1 to disable colors in the REPL\n"
+         "NODE_EXTRA_CA_CERTS      path to additional CA certificates file\n"
 #if defined(NODE_HAVE_I18N_SUPPORT)
          "NODE_ICU_DATA            data path for ICU (Intl object) data\n"
 #if !defined(NODE_HAVE_SMALL_ICU)
          "                         (will extend linked-in data)\n"
 #endif
+#ifdef _WIN32
+         "NODE_PATH                ';'-separated list of directories\n"
+#else
+         "NODE_PATH                ':'-separated list of directories\n"
+#endif
+         "                         prefixed to the module search path\n"
 #endif
          "NODE_REPL_HISTORY        path to the persistent REPL history file\n"
+         "NODE_TTY_UNSAFE_ASYNC    set to 1 to enable async stdout and stderr\n"
```

Comment:
Hmm, I feel it has it's place here, even if discouraged. The `UNSAFE` should probably make it clear enough that it has a caveat and users can look up details in the man page or in the online docs.